spec/controllers/rentals_controller_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe RentalsController, :type => :controller do
+
+  describe 'GET #index' do
+    let!(:rentals) { create_list(:rental, 10) }
+    
+    it 'returns a success response' do
+      get :index, params: {}
+      expect(response).to have_http_status(:success)
+      expect(JSON.parse(response.body)["data"].length).to eq(10)
```

Comment:
In the webapp, we use [this gem](https://github.com/collectiveidea/json_spec) for testing our JSON API responses.